x-pack/plugins/fleet/server/types/models/preconfiguration.ts

```diff
@@ -23,7 +23,13 @@ const varsSchema = schema.maybe(
     schema.object({
       name: schema.string(),
       type: schema.maybe(schema.string()),
-      value: schema.maybe(schema.oneOf([schema.string(), schema.number()])),
+      value: schema.maybe(
```

Comment:
I wonder if it makes sense to loosen this schema a bit, for example a boolean wouldn't pass this validation. looking at `ConfigRecordSchema`, (the equivalent schema for package policy objects), we use `value: schema.maybe(schema.any())` there

Reply:
Sounds good, let's do it